src/controller.js

```diff
@@ -0,0 +1,51 @@
+import Server from './server';
+import View from './views';
+
+export default class Controller {
+  static saveToLocal(response) {
+    return localStorage.setItem('response', JSON.stringify(response));
+  }
+
+  static getFromLocal() {
+    const obj = JSON.parse(localStorage.response);
+    return obj;
+  }
+
+  static reset() {
+    const results = document.getElementById('results');
+    results.innerHTML = '';
+    document.body.style.backgroundColor = 'ghostwhite';
```

Comment:
- [ ] [OPTIONAL] Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) (screenshot from Pathwright)

Reply:
added to future improvemnt. ✔️